code/modules/projectiles/guns/gun_parts.dm

```diff
@@ -15,22 +15,26 @@
 	name = "silencer"
 	desc = "Suppresses the muzzle report of the weapon, making it harder to detect visibly and audibly."
 	icon_state = "silencer"
+	var/mute_multi = 5 //if the number is bigger than 1, the gunshot sound is quiter, works the other way around too
```

Comment:
mute_multi is a really bad variable name. Also

```suggestion
	var/mute_multi = 5 //if the number is bigger than 1, the gunshot sound is quieter, works the other way around too
```

If multi means multiplicator then just type multiplicator